app/src/main/java/com/kickstarter/features/search/ui/SearchAndFilterActivity.kt

```diff
@@ -80,43 +87,46 @@ class SearchAndFilterActivity : ComponentActivity() {
 
                 val darModeEnabled = this.isDarkModeEnabled(env = env)
                 KickstarterApp(useDarkTheme = darModeEnabled) {
-                    SearchScreen(
-                        environment = env,
-                        onBackClicked = { onBackPressedDispatcher.onBackPressed() },
-                        scaffoldState = rememberScaffoldState(),
-                        errorSnackBarHostState = snackbarHostState,
-                        isLoading = isLoading,
-                        isDefaultList = currentSearchTerm.isTrimmedEmpty(),
-                        itemsList = if (currentSearchTerm.isTrimmedEmpty()) {
-                            popularProjects
-                        } else {
-                            searchedProjects
-                        },
-                        lazyColumnListState = lazyListState,
-                        showEmptyView = !isLoading && (searchedProjects.isEmpty() && popularProjects.isEmpty()),
-                        categories = categories,
-                        onSearchTermChanged = { searchTerm ->
-                            currentSearchTerm = searchTerm
-                            viewModel.updateSearchTerm(searchTerm)
-                        },
-                        onItemClicked = { project ->
-                            val projAndRef = viewModel.getProjectAndRefTag(project)
-                            if (project.displayPrelaunch().isTrue()) {
-                                startPreLaunchProjectActivity(project, projAndRef.second)
+                    CompositionLocalProvider(LocalFilterMenuViewModel provides filterMenuViewModel) {
```

Comment:
Notice here, we add to local composition the VM that controls the filterterMenu network requests